spec/acceptance/rspec_expectation_test_framework_integration_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'acceptance_spec_helper'
+
+describe 'shoulda-matchers integrates libs for rspec-expectations framework' do
```

Comment:
This file would be renamed/reframed to represent Cucumber and would set up a Cucumber project.